uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/Page/AbsenceType.php

```diff
@@ -183,4 +186,21 @@ private function canNotDelete($absenceType) {
     return $this->absenceTypeService->absenceTypeHasEverBeenUsed($absenceType->id)
            || $absenceType->is_reserved;
   }
+
+  /**
+   * Retrieves the category label from option value
+   *
+   * @param int $categoryId
+   *
+   * @return mixed
```

Comment:
Should this not just return a string?

Reply:
It used mixed because empty string was not explicitly return if key does not exist.